CHANGELOG/CHANGELOG-1.3.md

```diff
@@ -1,6 +1,8 @@
 Note that we start to track changes starting from v1.3.7.
 
-## v1.3.9(TBD)
+<hr>
+
+## v1.3.9(2024-02-24)
```

Comment:
Should we add a new TBD entry for `1.3.10`?  Or will we not have any more patch releases for this branch?

Reply:
We can add that when we have any new fix for 1.3.10. Most likely we won't have any fix for 1.3.10 in near future.